superbench/benchmarks/micro_benchmarks/cublaslt_function.py

```diff
@@ -21,8 +21,8 @@ def __init__(self, name, parameters=''):
         """
         super().__init__(name, parameters)
 
-        self._bin_name = 'cublaslt_fp8_gemm'
-        self._in_types = ['fp16', 'fp8e4m3', 'fp8e5m2']
+        self._bin_name = 'cublaslt_gemm'
+        self._in_types = ['fp64', 'fp32', 'fp16', 'bf16', 'fp8e4m3', 'fp8e5m2']
```

Comment:
For fp64 and fp32, are they TC operation or not?

Reply:
yes, all precisions will use tensor cores (fp32 is input type, compute type is tf32)